db/seeds.rb

```diff
@@ -39,5 +44,43 @@
 
 # Create Aircrafts
 aircrafts.each do |aircraft_data|
-  Aircraft.create!(aircraft_data)
+  aircraft = Aircraft.create!(aircraft_data)
+  1000.times do
```

Comment:
Might be nice for this to be a constant so we can easily ramp up or down the order of magnitude of the seeding.